aspnetcore/tutorials/first-web-api/sample/TodoApi/Startup.cs

```diff
@@ -28,10 +28,8 @@ public Startup(IHostingEnvironment env)
         // using Microsoft.EntityFrameworkCore;
         public void ConfigureServices(IServiceCollection services)
         {
-            // requires using Microsoft.EntityFrameworkCore;
             services.AddDbContext<TodoContext>(opt => opt.UseInMemoryDatabase());
 
-            // Add framework services.
             services.AddMvc();
             
             services.AddSingleton<ITodoRepository, TodoRepository>();
```

Comment:
Is this the same thing that there was an email thread about? It's hard to see a case where a singleton DbContext would be safe.